common/log.c

```diff
@@ -61,8 +62,15 @@ internal_log_file_open(const char *fname)
 
     if (fname != NULL)
     {
-        ret = open(fname, O_WRONLY | O_CREAT | O_APPEND | O_SYNC,
-                   S_IRUSR | S_IWUSR);
+	if (g_strcmp(fname, "-") != 0)
```

Comment:
`-` is a common symbol to represent standard out in command line arguments. However, this looks a little bit odd because the context is not a command line.

What about using a word "stdout" instead of `-`.

```
Logfile=stdout
```

Reply:
`stdout` looks like legitimate file name, an this is very unlikely that somebody will want to name log file with a single dash (`-`) name. This is why I've chosen this name.

If you fill it looks a little bit odd, what about `<stdout>`, with angle braces around the word?

And in any case, I consider it as a debug mode, not intended for the common user.

Reply:
I personally like `<stdout>`.